Week2task9/src/ClickOnButton.java

```diff
@@ -0,0 +1,31 @@
+import java.awt.FlowLayout;
+import java.awt.Frame;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import javax.swing.JButton;
+import javax.swing.JFrame;
+import javax.swing.JOptionPane;
+
+public class ClickOnButton extends JFrame  implements ActionListener {
+	
+	public ClickOnButton() {
+		super( "Learn IT Girl" );
+		setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+		setSize(200, 200);
+		setLocation(400, 200);
+		setLayout(new FlowLayout(70, 50, 50));
+		JButton b = new JButton("Click me!");
+		b.addActionListener(this);
+		add(b);
+		setVisible(true);
+	}
+	@Override
+	public void actionPerformed(ActionEvent e) {
+		// TODO Auto-generated method stub
+		// show a dialog with the message
+		int numClicks = 0;
```

Comment:
If you declare here the numClicks variable and initialise it at 0, each time you click on the button you will see 'You have clicked the button 1 times'. You should declare the numClicks variable somewhere line 10 and then in the constructor initialise it at 0. Here in the actionPerformed we just increment it.